src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -98,6 +98,7 @@ public function onKernelResponse(FilterResponseEvent $event)
             || $response->isRedirection()
             || ($response->headers->has('Content-Type') && false === strpos($response->headers->get('Content-Type'), 'html'))
             || 'html' !== $request->getRequestFormat()
+            || false !== strpos($response->headers->get('Content-Disposition'), 'attachment')
```

Comment:
Updated accordingly. The question is now the check, I am now checking for `attachment;` instead.